### DISCUSSION TITLE
Can I configure the filename of the Home page?

### DISCUSSION THREAD

**Discussion by Lost-Projects** [zim-desktop-wiki/zim-desktop-wiki#1642] [Q&A] [answered]
Hi,
A minor item: I'm migrating my filing system to Zim - basically to hyperlink text files. My main folders are numbered and I'm using the home page as the contents/index. To get the home page at the top of the list in the side pane, I numbered it as "0.Home". I don't tend to use the home button, but once deep in the folders, I realised I could click that button rather than scrolling up. But of course it doesn't recognise "0.Home" and creates a "Home" page. I'm currently going with that, but the home page can then drop down the listing, below all the numbered folders and sub-entries. I suppose I should just get used to the home button to jump, but my ideal solution would be to change a setting so that the default "Home" page can be customised - to "0.Home" in my case.

Is this possible in a configuration file somewhere?

Thanks for your time and for Zim-wiki.

**Comment by jaap-karssenberg** [comment, marked as answer]
Yes, please see the notebook properties in the "file" menu.

**Comment by Lost-Projects** [comment]
Brilliant! And easier than I expected.
Thanks again.
